### PR TITLE
interlock --> ucp-interlock

### PR DESCRIPTION
**Pull request by jcantosz**
### Proposed changes

The service is called `ucp-interlock` now, not interlock. This makes it more consistent with the rest of the docs.

### CODE REVIEW

**Comment by bermudezmt** [comment]
@david-yu @ehazlett can I have one of you take a quick look for accuracy?

File: ee/ucp/interlock/config/updates.md [hunk 1]
```diff
@@ -30,6 +30,6 @@ performs a rolling deploy to update all extensions.
 
 ```bash
 $> docker service update \
-    --image interlockpreview/interlock@sha256:d173014908eb09e9a70d8e5ed845469a61f7cbf4032c28fad0ed9af3fc04ef51 \
-    interlock
+    --image docker/ucp-interlock@sha256:d173014908eb09e9a70d8e5ed845469a61f7cbf4032c28fad0ed9af3fc04ef51 \
```
Comment (ehazlett) [inline ee/ucp/interlock/config/updates.md]: This digest is wrong as it doesn't exist for `docker/ucp-interlock`.  The latest image for 3.1.6 is `docker/ucp-interlock@sha256:9ba5e9ffd451b732d0e442fe8e5dc482580a196a8c98218f742b0d4395504854`.

**Comment by ehazlett** [comment]
Minor nit on the digest otherwise looks good.  Thanks @jcantosz

File: ee/ucp/interlock/config/updates.md [hunk 2]
```diff
@@ -20,16 +20,70 @@ $> docker config create service.interlock.conf.v2 <path-to-new-config>
...
 ```bash
-$> docker service update --config-rm service.interlock.conf interlock
-$> docker service update --config-add source=service.interlock.conf.v2,target=/config.toml interlock
+$> docker service update --config-rm service.interlock.conf ucp-interlock
+$> docker service update --config-add source=service.interlock.conf.v2,target=/config.toml ucp-interlock
 ```
 
-Next, update the Interlock service to use the new image. The following example updates the Interlock core service to use the `sha256:d173014908eb09e9a70d8e5ed845469a61f7cbf4032c28fad0ed9af3fc04ef51`
-version of Interlock. Interlock starts and checks the config object, which has the new extension version, and 
+Next, update the Interlock service to use the new image. To pull the latest version of UCP, run the following:
```
Comment (bermudezmt) [inline ee/ucp/interlock/config/updates.md]: @ehazlett please have a final look. Thanks again!

**Comment by ehazlett** [comment]
LGTM

**Comment by bermudezmt** [comment]
Thank you so much, @jcantosz and @ehazlett !

**Pull request merged**